### ISSUE TITLE
Add support for non-vibrating hardware

### ISSUE THREAD

**Issue by qdot** [intiface/intiface-game-haptics-router#1]
Would be nice to support the Fleshlight Launch, Kiiroo Onyx series, Vorze, etc.

**Comment by jos-s** [comment]
Yes that would be awesome!

**Comment by jos-s** [comment]
@qdot Personally I have got the Launch, I am guessing it is the most popular? Might get the Vorze some time in the future. Also have the Edge, which I think is great to use with the game vibration router/ intiface game haptics router. Launch is great, but can only use it with the scriptplayer so far. Would've been superb to have the launch compatible with the game haptics router aswell.  
Do you think it would be tricky to implement as vibrating and stroking are quite different?
Really like your work, keep doing what you're doing :)

**Comment by popeyeotaku** [comment]
Exactly what would be needed to implement this? I assume there's issues converting between rumble and linear automatically?

**Comment by qdot** [comment]
@popeyeotaku Yup. basically there'd need to be a way to translate from vibration to oscillation frequency, with ramp-up/ramp-down. It's something I've been vaguely poking at in the background for a while.

It doesn't even really need to be specific to the GHR, it's just an algorithm I'd like to ship as part of a utility library with our client implementations.
